cmd/expired-authz-purger/main.go

```diff
@@ -30,6 +30,14 @@ type eapConfig struct {
 		BatchSize   int
 		MaxAuthzs   int
 		Parallelism uint
+		// PendingCheckpointFile is the path to a file which is used to store the
+		// last pending authorization ID which was deleted. If path is to a file
+		// which does not exist it will be created.
+		PendingCheckpointFile string
+		// FinalCheckpointFile is the path to a file which is used to store the
+		// last pending authorization ID which was deleted. If path is to a file
```

Comment:
This comment says "the last pending authorization ID which was deleted" but I think it should be "the last **final** authorization which was deleted"